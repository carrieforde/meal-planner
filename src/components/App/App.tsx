function App() {
  return <div className="App">Add your components here!</div>;
}

export default App;
