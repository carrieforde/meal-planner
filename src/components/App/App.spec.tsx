import { render } from "@testing-library/react";
import App from "./App";

describe("App", () => {
  it("should render the App component consistently", () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
