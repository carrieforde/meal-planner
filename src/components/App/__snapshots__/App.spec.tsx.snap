// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the App component consistently 1`] = `
<div>
  <div
    class="App"
  >
    Add your components here!
  </div>
</div>
`;
